tests/test_views.py

```diff
@@ -2,6 +2,8 @@
 
 from flasql import views
 
+from tests.fixtures import app # noqa
```

Comment:
Ah, right, you have to explicitly say you're going to use it via `pytest.mark.usefixtures`, or put it in conftest.py. Not a big deal for now :)